<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnumDetails</class>
 <widget class="QWidget" name="EnumDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnumValue</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="enumContents">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="_EnumValueContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>396</width>
          <height>204</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="EnumValueContents"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="enumDeleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="valueAddButton">
       <property name="text">
        <string>Add Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
